### DISCUSSION TITLE
city/guangzhou

### DISCUSSION THREAD

**Discussion by giscus** [freeCodeCamp-China/city#45] [General]
# city/guangzhou

欢迎来到 freeCodeCamp 中文社区！与编程爱好者们一起探索全国不同城市的线下技术社区，从技术交流到结对编程，与我们一起分享，共同成长。

https://fcc-china.vercel.app/city/guangzhou

**Comment by luhuadong** [comment]
怎么加入本地社群？

**Reply by TechQuery** [reply]
联系方式详见本地城市社区主页首屏的社交平台图标链接。
